package.json

```diff
@@ -73,14 +73,18 @@
  },
  "dependencies": {
   "@sendgrid/mail": "^7.2.1",
+  "@types/config": "0.0.36",
   "@types/express": "^4.17.7",
   "@types/jsonwebtoken": "^8.5.0",
   "@types/ws": "^7.2.6",
   "class-validator": "^0.12.2",
+  "config": "^3.3.1",
   "dotenv": "^8.2.0",
+  "email-validator": "^2.0.4",
   "express": "^4.17.1",
   "jsonwebtoken": "^8.5.1",
   "pg": "^8.3.0",
+  "quickemailverification": "^1.0.3",
```

Comment:
Could you remove all the new dependencies that have been added since they aren't used?